backend/composer/management/commands/ingest_neurondm_new_field.py

```diff
@@ -0,0 +1,48 @@
+import time
+
+from django.core.management.base import BaseCommand
+from composer.services.cs_ingestion.neurondm_new_field_ingestion_service import ingest_neurondm_new_field_to_statements
+from composer.enums import NeurondmIngestionMapping
+
+
+class Command(BaseCommand):
+    help = "Ingests Statement's fields from neurondm pyp package"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--full_imports',
+            nargs='*',
+            help='List of full imports to include in the ingestion.',
+        )
+        parser.add_argument(
+            '--label_imports',
+            nargs='*',
+            help='List of label imports to include in the ingestion.',
+        )
+        parser.add_argument(
+            '--cs_field',
+            required=True,
+            help='The field name in ConnectivityStatement to update.',
+        )
+
+    def handle(self, *args, **options):
+        full_imports = options['full_imports']
+        label_imports = options['label_imports']
+        cs_field = options['cs_field']
+
+        try:
+            neurondm_field = NeurondmIngestionMapping.get_neurondm_field(
```

Comment:
issue: I don't believe we need the NeurondmIngestionMapping enum. I was expecting us to use [hasattr](https://www.w3schools.com/python/ref_func_hasattr.asp) for example. This way we don't limit ourselves to only the properties in the enum.